### DISCUSSION TITLE
My Game is broken

### DISCUSSION THREAD

**Discussion by MemTheBun** [DEMOLITIONDON96/Demolition-Engine#16] [Lua Q&A]
sus

**Comment by ThatOneSillyGuyDon** [comment]
can you show a screenshot? i'm confused on what you just said

**Comment by MemTheBun** [comment]
![image](https://user-images.githubusercontent.com/92312822/162510385-d4fa6f6c-a02c-4502-b546-532dc4b14867.png)
@DEMOLITIONDON96

**Reply by ThatOneSillyGuyDon** [reply]
> ![image](https://user-images.githubusercontent.com/92312822/162510385-d4fa6f6c-a02c-4502-b546-532dc4b14867.png)
> @DEMOLITIONDON96

But the screen isn’t upside down and stage looks fine?

**Reply by ghost** [reply]
He said the app window went down when downscroll was enabled.

**Reply by MemTheBun** [reply]
downscroll is disabled

**Reply by ThatOneSillyGuyDon** [reply]
> downscroll is disabled

Weird, it works fine for me

**Reply by ghost** [reply]
> downscroll is disabled

Oh. I think this is outdated anyway.
